### DISCUSSION TITLE
Markdown formatting support

### DISCUSSION THREAD

**Discussion by harefoot55** [codecadwallader/codemaid#914] [Ideas]
Would it be possible to add document formatting support (text wrapping) for Markdown documents in Visual Studio? Thanks for continuing to develop these projects for the community.

![screencap_20220310_112028](https://user-images.githubusercontent.com/16060436/158901283-d2ee1c5b-ba69-4e3d-8b28-199fdb782a89.png)

**Comment by codecadwallader** [comment]
Thanks for suggesting the idea.  We currently only support text wrapping at a column for code comments.  As an awkward workaround I suspect the markdown content could be dropped into a code comment, formatted to cause the wrapping, and then moved back to the original markdown file.  I think that would only work for very basic plain markdown like the screenshot above.  If there were more markup indicators I'm not sure if they would still work correctly when wrapped.  As a possible example if there was a header line that was 81 characters long I don't think it would still look correct if extra characters were wrapped to the next line.
